tests/apps/mip/snapshots/snap_test_apps_mip_files.py

```diff
@@ -6,44 +7,46 @@
 
 snapshots = Snapshot()
 
-snapshots["test_parse_sampleinfo_rna_result_contents 1"] = {
-    "bcftools_merge": "/path/to/stuff/rare-disease/cases/case_id/analysis/files/case_id_comb.vcf",
-    "case": "case_id",
-    "config_file_path": "/path/to/stuff/rare-disease/cases/case_id/analysis/case_id_config.yaml",
-    "date": GenericRepr("datetime.datetime(2019, 11, 21, 14, 31, 2)"),
-    "is_finished": True,
-    "multiqc_html": "/path/to/stuff/rare-disease/cases/case_id/analysis/files/multiqc_report.html",
-    "multiqc_json": "/path/to/stuff/rare-disease/cases/case_id/analysis/files/multiqc_data.json",
-    "pedigree_path": "/path/to/stuff/rare-disease/cases/case_id/analysis/files/case_id.fam",
-    "qcmetrics_path": "/path/to/stuff/rare-disease/cases/case_id/analysis/files/qcmetrics_file.stub",
-    "samples": [
+snapshots['test_parse_sampleinfo_rna_result_contents 1'] = {
```

Comment:
Myabe black it again?

Reply:
I'll black the whole thing again when I'm done :)

Reply:
Make sense